packages/babel-cli/src/babel/util.js

```diff
@@ -111,3 +112,34 @@ export function adjustRelative(relative, keepFileExtension) {
   }
   return relative.replace(/\.(\w*?)$/, "") + ".js";
 }
+
+export function printSettings() {
+  const hasYarn = cwd =>
```

Comment:
should babel be responsible for checking yarn / npm / node versions? Seems a little bit odd, would ease copying "complete" environment setup to the issues though, so i dont know 🤷